e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -189,12 +188,80 @@ public async Task RegistryManager_AddDeviceWithProxy()
             await registryManager.AddDeviceAsync(device).ConfigureAwait(false);
         }
 
+        [LoggedTestMethod]
+        public async Task RegistryManager_ConfigurationOperations_Work()
+        {
+            // arrange
+
+            bool configCreated = false;
+            string configurationId = (_idPrefix + Guid.NewGuid()).ToLower(); // Configuration Id characters must be all lower-case.
+            using RegistryManager client = RegistryManager.CreateFromConnectionString(TestConfiguration.IoTHub.ConnectionString);
+
+            try
+            {
+                var expected = new Configuration(configurationId)
+                {
+                    Priority = 2,
+                    Labels = { { "labelName", "labelValue" } },
+                    TargetCondition = "*",
+                    Content =
+                    {
+                        DeviceContent = { { "properties.desired", "\"setting1\": 4" } },
```

Comment:
I'm unsure on what this is supposed to be. As a payload to test, it works fine because GIGO is fine for testing the SDK, but I'd rather this be a logical payload that a customer might use.

Anyone have an idea here?

Reply:
I do not know either.

Reply:
Rick Kessler may be able to fill in the blanks here. I reached out to him on Teams and waiting for his availability today.

Reply:
Rick helped me. I feel ramped up now. :)